src/Sulu/Bundle/AdminBundle/Resources/js/services/Router/Router.js

```diff
@@ -39,7 +39,7 @@ function tryParse(value: ?string) {
         }
     }
 
-    if (isNaN(value)) {
```

Comment:
would prefer to do a `value && value.startsWith('0')` in a separate check. also it would be great to add a test to the `Router.test.js` to make sure that this will not be changed in the future 🙂

Reply:
Using `value.startsWith('0')` will make any float that starts with 0 return as a string, I think that might create some problems with strict comparisons, that's why I used a regular expression. I think we could skip the `isNaN(value)` check and always return as a string at that point if we changed to loose comparisons but that might cause some problems on its own.

I'll try to write some tests on monday and make the check separately.

Thanks for the feedback!

Reply:
you are right - thanks for thinking this through 🙂